plugins/github/server/github.ts

```diff
@@ -229,11 +230,20 @@ export class GitHub {
       return;
     }
 
+    // Find integration, prioritizing one where the installation account matches the resource owner
     const integration = (await Integration.findOne({
       where: {
         service: IntegrationService.GitHub,
         teamId: actor.teamId,
       },
+      order: [
+        [
+          Sequelize.literal(
```

Comment:
Too dangerous to inject the owner into a literal like this. If we can't use replacements then I'd rather query all the integrations and match in js